bin/sdk/gutenberg.js

```diff
@@ -37,7 +39,7 @@ exports.compile = args => {
 			},
 			output: {
 				path: options.outputDir,
-				filename: `${ name }-editor.js`,
+				filename: '[name]',
```

Comment:
This will also make output files end in `-script.js` - I think that's unnecessary, what do you think about removing it?

Reply:
removed in f53973c